src/base.config.ts

```diff
@@ -217,7 +225,16 @@ export default function webpackConfigFactory(args: any): WebpackConfiguration {
 							options: { features: args.features }
 						},
 						getUMDCompatLoader({ bundles: args.bundles }),
-						{ loader: 'ts-loader', options: { onlyCompileBundledFiles: true, instance: 'dojo' } }
+						{
+							loader: 'ts-loader',
+							options: {
+								onlyCompileBundledFiles: true,
+								instance: 'dojo',
+								getCustomTransformers: () => ({
+									before: [registryTransformer(basePath, lazyModules)]
```

Comment:
might be nice to only add this if we have `lazyModules`? not sure of the performance costs (we do not exit early from the transformer)

Reply:
Ah yeah that's a good suggestion.